framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/wwwroot/libs/abp/aspnetcore-mvc-ui-theme-shared/datatables/datatables-extensions.js

```diff
@@ -231,7 +231,7 @@
     (function () {
         datatables.createAjax = function (serverMethod, inputAction) {
             return function (requestData, callback, settings) {
-                var input = inputAction ? inputAction() : {};
+                var input = inputAction ? inputAction(requestData) : {};
```

Comment:
```suggestion
                var input = inputAction ? inputAction(requestData, settings) : {};
```

Maybe we can pass `settings` together .

Reply:
Good idea!